RiotSwiftUI/Modules/Room/Composer/Test/UI/ComposerUITests.swift

```diff
@@ -45,6 +45,33 @@ final class ComposerUITests: MockScreenTestCase {
         XCTAssertTrue(maximiseButton.exists)
     }
     
+    // This test requires "connect hardware keyboard" to be off on the simulator
+    // And may not work on the CI
+//    func testFastTyping() throws {
+//        app.goToScreenWithIdentifier(MockComposerScreenState.send.title)
+//        let text = "fast typing test"
+//        let wysiwygTextView = app.textViews.allElementsBoundByIndex[0]
+//        XCTAssertTrue(wysiwygTextView.exists)
+//        wysiwygTextView.tap()
+//        sleep(2)
+//        wysiwygTextView.typeText(text)
+//        let value = wysiwygTextView.value as! String
+//        XCTAssert(value == text, "Text view value is: \(value)")
+//    }
```

Comment:
sadly this test works locally but not on the CI, this is either because the CI uses the hardware keyboard, or is just too slow to test efficiently typing fast text.

Reply:
Indeed, the CI is the reason why we have this thing actually: 

`private extension XCUIElement {
    /// Types a text inside the UI element character by character.
    /// This is especially useful to avoid missing some characters on
    /// UI tests running on a rather slow CI.
    ///
    /// - Parameters:
    ///   - text: Text to type in the UI element.
    func typeTextCharByChar(_ text: String) {
        text.forEach { self.typeText(String($0)) }
    }
}`

I wonder if we can come up with something smart that expects this test to fail on CI, but succeed on most local machines, rather than just commenting 🤔 
Adding something like this before your failing assertion will not block ever, but at least we'll try to run the test and could see how it evolves: 

`let options = XCTExpectedFailure.Options()
    options.isStrict = false
    XCTExpectFailure("Test may or may not fail on CI", options: options)`